### ISSUE TITLE
[Question] Copyleft

### ISSUE THREAD

**Issue by dannycolin** [raisely/NoHarm#12]
Is there a reason for not using a copyleft license?

**Comment by dannycolin** [comment]
I've read it and I didn't see the reasons for not using a copyleft license vs. a more permissive license. 

I reread the license and the point 5 state that : 

 > All redistribution of source code or binary form, including any modifications must be under these terms. You must inform recipients that the code is governed by these conditions, and how they can obtain a copy of this license. You may not attempt to alter the conditions of who may/may not use this software.

So it isn't a BSD 3-clause anymore but an extensively modified one. If developers need to put their modification under the same license then it is kind of a copyleft license. Maybe you should say that in the README.md.

**Comment by chrisjensen** [comment]
Ah, sorry. I misinterpreted your questions as why not use a copyleft license _instead_ of this license (as opposed to as the basis of the license).

We were striving to keep the license simple to understand, and the (L)GPL & MPL licenses are quite long and in some cases hard to parse for most people, so we felt building on those would've made for a harder (and less adoptable) license.

Good point about the README.

**Comment by dannycolin** [comment]
> We were striving to keep the license simple to understand, and the (L)GPL & MPL licenses are quite long and in some cases hard to parse for most people, so we felt building on those would've made for a harder (and less adoptable) license.

Yes, It's true that they're a bit hard to parse at first. :)

Last question, have you consider to add a clause that cover Software as a Service. Since the software isn't distributed, someone can avoid distributing his/her modifications if the software is runnning (hosted) on a remote computer. That's why the FSF created the AGPLv3.

A solution could be to create a JWL and a JWL+ License if you feel that it wouldn't be a good idea to enforce that clause. But at least, people would have the choice to add it or not. What do you think about that?

**Comment by chrisjensen** [comment]
So prohibiting it linking to code (via a web API) that is in breach of the JWL?

**Comment by chrisjensen** [comment]
#14

**Comment by chrisjensen** [comment]
I'm going to close this for now, but happy to re-open if there are more comments.
